io.github.ra3xdh.qucs_s.yml

```diff
@@ -0,0 +1,48 @@
+id: io.github.ra3xdh.qucs_s
+runtime: org.kde.Platform
+runtime-version: 6.8
+sdk: org.kde.Sdk
+command: qucs-s
+rename-desktop-file: qucs-s.desktop
+rename-icon: qucs-s
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --filesystem=host
```

Comment:
why is host needed?

Reply:
The app requires this permission to save files.

Reply:
It does support portals, but without filesystem permissions, the file won’t be saved correctly, similar to https://github.com/flathub/de.bund.ausweisapp.ausweisapp2/issues/2

Reply:
I guess that doesn't go through a filechooser? Otherwise it should work.

Also why do you need host? Log files aren't usually saved to external disks.

Reply:
I'm not the developer, so I'm not familiar with the reason behind this. I'll go ahead and change it to `--filesystem=home`.